src/topp/IDMerger.cpp

```diff
@@ -356,7 +363,9 @@ class TOPPIDMerger :
         }
         peptides.insert(peptides.end(), base_peptides.begin(),
                         base_peptides.end());
-        // merge in data from other files:
+        ///cut all other PeptideIdentifications to the first hit after sorting
+        /// and add new sequences, add the first ProteinHits corresponding to
+        /// the new sequence
```

Comment:
Basically, adding IDMerger to the PR was rather an accident. I was looking at the tool and to better understand it's workings i thought I could add comments.